pkg/provider/keycloak/keycloak.go

```diff
@@ -393,6 +393,8 @@ func updateKeyCloakFormData(authForm url.Values, s *goquery.Selection, user *cre
 		authForm.Add(name, user.Username)
 	} else if strings.Contains(lname, "password") {
 		authForm.Add(name, user.Password)
+	} else if strings.Contains(lname, "tryanotherway") {
```

Comment:
If the intention is to ignore the case, maybe we should log that as the intention?

Reply:
I think logging the information will produce more noice than provide any relevant information to the user. Therefore i haven't added any handling besides ignoring this case.

Reply:
If the "Try another Way" button is present, the keycloak admin has configured other methods to login.
These are effectively ignored when adding this code, which is totally fine for now (in my opinion).
Nevertheless, the ultimate goal should be adding the possibility to use the other login methods.
Therefore, a small message like: "Ignoring other ways to log in (not implemented)" would be great... Also for the next person who has to debug this ;)

Reply:
Added debug log that other options are currently not implemented